src/pages/home/report/ReportActionCompose/index.js

```diff
@@ -741,214 +746,220 @@ class ReportActionCompose extends React.Component {
                 this.props.isComposerFullSize && styles.chatItemFullComposeRow,
             ]}
             >
-                {shouldShowReportRecipientLocalTime && <ParticipantLocalTime participant={reportRecipient} />}
-                <View style={[
-                    shouldUseFocusedColor
-                        ? styles.chatItemComposeBoxFocusedColor
-                        : styles.chatItemComposeBoxColor,
-                    styles.flexRow,
-                    styles.chatItemComposeBox,
-                    this.props.isComposerFullSize && styles.chatItemFullComposeBox,
-                    hasExceededMaxCommentLength && styles.borderColorDanger,
-                ]}
+                <OfflineWithFeedback
+                    pendingAction={this.props.pendingAction}
```

Comment:
Coming from this issue https://github.com/Expensify/App/issues/44572, adding pendingAction here will grey out the composer for the newly created report, which is not the expected result. The composer should not be greyed out. More context is provided in this comment: https://github.com/Expensify/App/issues/44572#issuecomment-2213150772